fontforge/python.c

```diff
@@ -2428,10 +2460,13 @@ return( NULL );
 	self->points[i] = c1->points[i];
     }
     for ( i=0; i<c2->pt_cnt; ++i ) {
-	Py_INCREF(c2->points[i]);
+	if ( c2!=&dummy )
+	    Py_INCREF(c2->points[i]);
 	self->points[c1->pt_cnt+i] = c2->points[i];
     }
-Py_RETURN( (PyObject *) self );
+    if ( ((PyObject *)c2)!=_c2 && c2!=&dummy )
```

Comment:
PyFFPoint_Parse grabs a reference while PyFFPointList_Parse doesn't grab references for the copied points, So we need to increment when copying from the latter or a passed contour but not from the "dummy" point. 
(The contour returned from PyFFPointList_Parse is really just a handy container for a point sequence.)